syserol/impute.py

```diff
@@ -131,7 +131,7 @@ def impute_accuracy(missingCube, missingGlyCube, comps, PCAcompare=True, ALS=Tru
 
     for ii, nComp in enumerate(comps):
         # reconstruct with some values missing
-        recon_cmtf = perform_CMTF(missingCube, missingGlyCube, nComp, ALS=ALS)
```

Comment:
With the missing pattern pre-computed, I don't think this will be all that slow anymore.